players/yuna.py

```diff
@@ -0,0 +1,26 @@
+from players.base import Player
+import logging
+import memory
+import xbox
+
+logger = logging.getLogger(__name__)
+
+class YunaImpl(Player):
+    
+    def __init__(self):
+        super().__init__("Yuna", 1, [0, 23, 22, 1])
+        
+    def overdrive(self, aeon_num: int = 0):
```

Comment:
Just as we are changing the players to named objects, maybe it'd be good to use names/constants for the Aeons as well - i.e. overdrive(bahamut) instead of overdrive(4). Maybe in the next pass.

Reply:
Agreed. A few of the aeons need to be implemented differently, so I'll add that into the next one.